client/src/box/flexBox.ts

```diff
@@ -3,4 +3,11 @@ import styled from 'styled-components'
 export const Wrapper = styled.div`

```

Comment:
I think it makes more sense to name `Flex` instead of `Wrapper`

Reply:
Ok, i renamed it